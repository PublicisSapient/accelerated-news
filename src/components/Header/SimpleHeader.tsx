import React from 'react';

export const SimpleHeader = () => {
  return (
    <header className="bg-primary-main px-2">
      <nav className="navbar">
        <span className="navbar__brand">Accelerated News</span>
      </nav>
    </header>
  );
};
