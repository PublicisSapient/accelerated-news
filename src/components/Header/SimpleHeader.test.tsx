import React from 'react';
import { render } from '@testing-library/react';
import { SimpleHeader } from './SimpleHeader';

describe('<SimpleHeader />', () => {
  test('renders correctly', () => {
    const { asFragment } = render(<SimpleHeader />);
    expect(asFragment()).toMatchSnapshot();
  });
});
