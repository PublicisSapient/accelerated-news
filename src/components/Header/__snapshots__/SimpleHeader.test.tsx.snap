// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimpleHeader /> renders correctly 1`] = `
<DocumentFragment>
  <header
    class="bg-primary-main px-2"
  >
    <nav
      class="navbar"
    >
      <span
        class="navbar__brand"
      >
        Accelerated News
      </span>
    </nav>
  </header>
</DocumentFragment>
`;
